test/data/inventories.sql

```diff
@@ -0,0 +1,2862 @@
+,(0xD81D1EE0727C11EA8241000C2997DDC0,'Petits Equipements Chirurgicau','E.SUR',242,201,162,NULL,0,1,0)
+,(0xD81D1FA7727C11EA8241000C2997DDC0,'Kit Laboratoire','K.LAB',242,201,162,NULL,0,1,1)
+,(0xD81D2067727C11EA8241000C2997DDC0,'Kits médicaux','K.MED',242,201,162,NULL,0,1,1)
+,(0xD81D212C727C11EA8241000C2997DDC0,'Kits Chirurgicaux','K.SUR',242,201,162,NULL,0,1,1)
+;
+INSERT INTO inventory_group (uuid,name,code,sales_account,cogs_account,stock_account,donation_account,unique_item,tracking_consumption,tracking_expiration) VALUES 
+(0xD81D21FA727C11EA8241000C2997DDC0,'Nutritions','N.FOS',242,201,162,NULL,0,1,1)
+,(0xD81D22BB727C11EA8241000C2997DDC0,'Equipement Chaine de froid','P.COL',242,201,162,NULL,0,1,0)
+,(0xD81D2385727C11EA8241000C2997DDC0,'Equipement Eléctrique','P.ELE',242,201,162,NULL,0,1,0)
+,(0xD81D244F727C11EA8241000C2997DDC0,'Sondes et Drains','S.CTD',242,201,162,NULL,0,1,1)
+,(0xD81D251A727C11EA8241000C2997DDC0,'Bandages et Pansements','S.DRE',242,201,162,NULL,0,1,1)
+,(0xD81D25DD727C11EA8241000C2997DDC0,'Matériels d''injection','S.INS',242,201,162,NULL,0,1,1)
+,(0xD81D26B2727C11EA8241000C2997DDC0,'Réactifs de laboratoire','S.LAS',242,201,162,NULL,0,1,1)
+,(0xD81D278D727C11EA8241000C2997DDC0,'Dispositifs de laboratoire','S.MST',242,201,162,NULL,0,1,1)
+,(0xD81D2857727C11EA8241000C2997DDC0,'Dispositifs Médico-chirurgicaux','S.MSU',242,201,162,NULL,0,1,1)
+,(0xD81D291E727C11EA8241000C2997DDC0,'Sutures et Ligatures','S.SUT',242,201,162,NULL,0,1,1)
+;
+INSERT INTO inventory_group (uuid,name,code,sales_account,cogs_account,stock_account,donation_account,unique_item,tracking_consumption,tracking_expiration) VALUES 
+(0xD81D29E8727C11EA8241000C2997DDC0,'Produits et Consommables de Radio','S.XRS',242,201,162,NULL,0,1,1)
+,(0xF4C4F0D8728411EA8241000C2997DDC0,'Eau, Hygiène et Assainissement','3167969',NULL,NULL,NULL,NULL,0,1,0)
+;
+
+-- 
+-- inventory
+-- 
+
+INSERT INTO inventory (enterprise_id,uuid,code,`text`,price,default_quantity,group_uuid,unit_id,unit_weight,unit_volume,stock,stock_max,stock_min,type_id,consumable,sellable,note,locked,delay,avg_consumption,purchase_interval,last_purchase,num_purchase,num_delivery,importance,created_at,updated_at) VALUES 
+(1,0x3CF8D982EEF911EA880100155D871701,'DORA_ERYT5T-_0','Erythromycine, 500mg, Tab, 500, Vrac',3.1800,1,0xD81D17A7727C11EA8241000C2997DDC0,18,0,0,0,0,0,1,1,1,NULL,0,1.0000,0.0000,0.0000,NULL,0,0,NULL,'2020-03-30 12:49:56',NULL)
+,(1,0x43F3DECBFCE9426E940ABC2150E62186,'DORA_QUIN1S-_0','Quinine Bichlorhydrate, sirop, 100mg base/5ml, 100ml, flacon, Unité',6.6300,1,0xD81D17A7727C11EA8241000C2997DDC0,18,0,0,0,0,0,1,1,1,NULL,0,1.0000,0.0000,0.0000,NULL,0,0,NULL,'2020-03-30 12:49:57',NULL)
+,(1,0x6B4825F14E6E47998A81860531281437,'DORA_PARA5TB_0','Paracetamol, 500mg, Tab, 1000, Blister',1.7400,1,0xD81D17A7727C11EA8241000C2997DDC0,18,0,0,0,0,0,1,1,1,NULL,0,1.0000,0.0000,0.0000,NULL,0,0,NULL,'2020-03-30 12:49:57',NULL)
```

Comment:
We should use relative dates in our tests to make sure we have accurate tests going forward that do not break with time 😄